src/deploy.ts

```diff
@@ -519,3 +517,16 @@ function getProjectContracts(): string[] {
 
   return deployableContracts;
 }
+
+/**
+ * Gets the latest commit hash
+ * @returns Commit hash
+ */
+function getLatestCommitHash(): string {
+  try {
+    return execSync("git rev-parse HEAD").toString().trim();
```

Comment:
Should work in CI but we should test before pushing a release

Reply:
Will definitely test before merging